themes/default/content/docs/reference/service-rest-api/_index.md

```diff
@@ -1679,6 +1679,247 @@ Status: 204 OK
 EMPTY RESPONSE BODY
 ```
 
+### Create Webhook
+
+```
+POST /api/orgs/{organization}/hooks
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `active` | boolean | body | enable webhook |
+| `displayName` | string | body | name of webhook |
+| `organizationName` | string | body | organization name |
+| `payloadUrl` | string | body | URL to send request to |
+| `secretName` | string | body | **Optional.** secret used as the HMAC key. See [webhook docs](/docs/intro/console/webhooks/#headers) for more information  |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+8' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token $PULUMI_ACCESS_TOKEN' \
+  --request POST \
+  --data '{"organizationName":"{organization}","displayName":"My Webhook","payloadUrl":"https://mywebhook.com","secret":"mysecret","active":true}' \
```

Comment:
```suggestion
  --data '{"organizationName":"{organization}","displayName":"My Webhook","payloadUrl":"https://example.com","secret":"mysecret","active":true}' \
```

`example.com` is a [reserved domain](https://en.wikipedia.org/wiki/Example.com) so I think it's best to use that rather than an arbitrary domain that might link to a "bad" website.